bibtex/gonzalez-fierro_conferences.bib

```diff
@@ -1,5 +1,12 @@
 # Conferences
 
+@inproceedings{joshi2022implementing,
+  title     = {Implementing a Parallel MLOps Test Pipeline for Open Source Development},
+  author    = {P. V. Joshi and A. Argyriou and M. Gonz{\'a}lez-Fierro},
```

Comment:
@pradnyeshjoshi this is a personal repo I use to track my publications. I would like to ask you how do you prefer to appear in the bibtex. Do you prefer `P. V. Joshi` or `P. Vineet Joshi` or other?

Reply:
@miguelgfierro `P. V. Joshi` works!